_docs/_latest/configure/scanner/rule-violations.md

```diff
@@ -15,4 +15,4 @@ file. To learn more, see
 
 **Below is an example of scanner violation output**
 
-{% responsive_image path: images/docs/quickstarts/scanner-output.png alt: "sample scanner output" %}
```

Comment:
This document doesn't seem to be describing how to configure anything. Can we evaluate it's need here or if we need it at all?

When we do we should also look at describing in detail all the configuration of violations, not their output. Output is considered 'use'.